### ISSUE TITLE
How to train this model by my own datasets?

### ISSUE THREAD

**Issue by skyloevil** [deeppomf/DeepCreamPy#35]
Can you share your train code , really appreciate to share them~

**Comment by pureexe** [comment]
I also want to try to train with my own dataset too.

**Comment by BroodWyrm** [comment]
Looks like this network doesn't work like that. It is some kind of autoencoder with partial convolution layers. It only use the source image for training.

**Comment by Sphinxs** [comment]
Nice feature (add your own datasets) ha ha

**Comment by Zzzzy411** [comment]
Well, I have the same problem..

**Comment by deeppomf** [comment]
Training code is from https://github.com/MathiasGruber/PConv-Keras

**Comment by deeppomf** [comment]
The training code with the modifications I made I don't intend to release.
I want all work to this project to be AGPL.
If I release the training code under AGPL, that doesn't mean neural network models generated from the training code are AGPL. It's like how works made with Blender, which a GPL 3D animation software, aren't GPL even though Blender is GPL.
Plus, the modifications I made are tailored to the dataset I have and wouldn't make sense with other datasets.
